setup.py

```diff
@@ -41,6 +41,8 @@ def run_tests(self):
     'urlcanon>=0.1.dev16',
     'doublethink>=0.2.0.dev81',
     'PySocks',
+    'warcio',
+    'requests'
```

Comment:
Dependencies that are required for tests but not for installation of the package should go in `tests_require`. As you can see below, `tests_require` already has `requests`. Only thing is that it's missing `warcio`.

Reply:
OK, I added `warcio` to `tests_require` and removed what I did in `install_requires`.